backend/src/routes/api/health/index.ts

```diff
@@ -0,0 +1,15 @@
+import { FastifyInstance, FastifyReply, FastifyRequest } from 'fastify';
+import { health } from './healthUtils';
+
+export default async (fastify: FastifyInstance): Promise<void> => {
+  fastify.get('/', async (request: FastifyRequest, reply: FastifyReply) => {
```

Comment:
Can we just get a comment here to say this is intentionally not secure so no one goes and secures it after the fact?